cjdns-snode/src/peer/info.rs

```diff
@@ -8,14 +10,18 @@ use crate::peer::{Peer, PeerList, Peers};
 pub struct PeersInfo {
     pub peers: Vec<PeerInfo>,
     pub announcements: usize,
+    #[serde(rename = "annByHashLen")]
     pub ann_by_hash_len: usize,
 }
 
 #[derive(Serialize)]
 pub struct PeerInfo {
     pub addr: String,
+    #[serde(rename = "outstandingRequests")]
     pub outstanding_requests: usize,
+    #[serde(rename = "msgsOnWire")]
     pub msgs_on_wire: usize,
+    #[serde(rename = "msgQueue")]
     pub msg_queue: usize,
 }
```

Comment:
Better add rename directive to every field, so that internal and external names are clearly saparated, and not affected by, for example, further refactorings.

Reply:
Done